www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -69,137 +62,6 @@
                 </div>
               <?php endif; ?>
 
-                <?php if ( !$current_member[HOUSE_TYPE_COMMONS] ) { ?>
```

Comment:
I think the commit message for this commit should explain more that it isn't just moving, it's removing non-MP voting records entirely (as far as I read it?) (Also need a space before my email address)

Reply:
Also should some of the 'default' section of www/docs/mp/index.php be removed now if it's no longer needed in this template? (that would impact upon my other comment)

Reply:
Ah yes - I was assuming we didn't have them but we do for the House of Lords sometimes or for ex-MPs who become something else. 

Will investigate the default seciton.